### ISSUE TITLE
The error handler doesn't work anymore

### ISSUE THREAD

**Issue by germsb** [vueneue/vueneue#19]
nothing happens when you access an invalid page or have a code / async error

**Comment by chymz** [comment]
Can you share you `src/App.vue` content ?

**Comment by germsb** [comment]
```
<template>
  <div id="app">
    <div id="nav">
      <router-link to="/">Home</router-link> |
      <router-link to="/about">About</router-link>
    </div>
    <router-view/>
  </div>
</template>

<style lang="scss">
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
}
#nav {
  padding: 30px;
  a {
    font-weight: bold;
    color: #2c3e50;
    &.router-link-exact-active {
      color: #42b983;
    }
  }
}
</style>
```

**Comment by chymz** [comment]
Fixed in `0.3.14`
